rds-postgres/replica/variables.tf

```diff
@@ -9,6 +9,12 @@ variable "apply_immediately" {
   default     = false
 }
 
+variable "ca_cert_id" {
+  type        = string
+  description = "Certificate authority for RDS database"
+  default     = "rds-ca-rsa2048-g1"
```

Comment:
When a new database is created on RDS, AWS already defines a default certificate authority. The default certificate currently is the same as we're defining here. However, if AWS changes that default certificate to something else, we will accidentally override the recommended default.

I think this is a rare case where we would just manually make the change on the console and not worry about reflecting that change on Terraform. Or, if we want to be able to override the certificate, I would think it would be better to not set a default here (what happens if we set the default to "null"? Would it pick RDS' default?).

What do you think, @OlamideOl1 @WilliamAvila?

Reply:
I raised https://github.com/thoughtbot/terraform-aws-databases/pull/29 for that, but I didn't have a chance to test what happens when creating a new database